stepfunctions_activity_worker/activity_worker.py

```diff
@@ -68,28 +64,47 @@ def perform_task(self):
         except (Exception, KeyboardInterrupt) as error:
             *_, raw_traceback = sys.exc_info()
             formatted_traceback = traceback.format_tb(raw_traceback)
-
-            print(f"{self.activity_name} failed!")
             self.stepfunctions.send_task_failure(
                 taskToken=task["taskToken"],
                 error=str(error)[:256],
                 cause="\n".join(formatted_traceback),
             )
 
             raise
+        return output
 
-        print(f"{self.activity_name} is completed!")
-        print(json.dumps(output, indent=4, sort_keys=True))
-        self.stepfunctions.send_task_success(
-            taskToken=task["taskToken"],
-            output=json.dumps(output, sort_keys=True),
-        )
+    def complete_task(self, future, task):
+        """Send the task results"""
+        error = future.exception()
+        if error:
+            print(f"{self.activity_name} failed!")
+        else:
```

Comment:
Unnecessary else block; we could probably return in the `if future.exception()` check.